dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -185,37 +189,50 @@ class LldpManager(object):
     def run(self):
         """
         Subscribes to notifications of changes in the PORT table
-        of the Redis State database. 
-        Subscribe to STATE_DB - get notified of the creation of an interface
+        of the Redis Config/Application database.
+        Subscribe to APP_DB - get port oper status
         Subscribe to CONFIG_DB - get notified of port config changes
         Update LLDP configuration accordingly.
         """
         # Set select timeout to 10 seconds
         SELECT_TIMEOUT_MS = 1000 * 10
 
-        # Subscribe to PORT table notifications in the State DB
         sel = swsscommon.Select()
-        sst = swsscommon.SubscriberStateTable(self.state_db, swsscommon.STATE_PORT_TABLE_NAME)
-        sel.addSelectable(sst)
-        sst = swsscommon.SubscriberStateTable(self.config_db, swsscommon.CFG_PORT_TABLE_NAME)
-        sel.addSelectable(sst)
 
-        # Listen for changes to the PORT table in the STATE_DB and CONFIG_DB
+        # Subscribe to PORT table notifications in the Config DB
+        sst_confdb = swsscommon.SubscriberStateTable(self.config_db, swsscommon.CFG_PORT_TABLE_NAME)
+        sel.addSelectable(sst_confdb)
+
+        # Subscribe to PORT table notifications in the App DB
+        sst_appdb = swsscommon.SubscriberStateTable(self.appl_db, swsscommon.APP_PORT_TABLE_NAME)
+        sel.addSelectable(sst_appdb)
+
+        # Listen for changes to the PORT table in the CONFIG_DB and APP_DB
         while True:
             (state, c) = sel.select(SELECT_TIMEOUT_MS)
 
             if state == swsscommon.Select.OBJECT:
-                (key, op, fvp) = sst.pop()
-
-                fvp_dict = dict(fvp)
-
-                # handle creation
-                if op == "SET" and fvp_dict.get("state") == "ok":
-                    self.generate_pending_lldp_config_cmd_for_port(key)
-
-                # handle config change
-                if op in ["SET", "DEL"] and (fvp_dict.get("alias") or fvp_dict.get("description")) :
-                    self.generate_pending_lldp_config_cmd_for_port(key)
+                (key, op, fvp) = sst_confdb.pop()
+                if fvp:
+                    fvp_dict = dict(fvp)
+
+                    # handle config change
+                    if op in ["SET", "DEL"] and (fvp_dict.get("alias") or fvp_dict.get("description")) :
```

Comment:
What if fvp_dict doesn't have such keys as alias or description?
That method will give us KeyError exception.
Should we better to check the keys in a way `
'alias' in fvp_dict or 'description' in fvp_dict`?

Reply:
Sure, I will first check if the keys are present and only then "get" them

---
In reply to: [304079960](https://github.com/Azure/sonic-buildimage/pull/3152#discussion_r304079960) [](ancestors = 304079960)

Reply:
I think this is fine, and doesn't need to be changed.